themes/vue/source/css/_font.styl

```diff
@@ -0,0 +1,59 @@
+@font-face {
+  font-family: Vazir;
+  src: url('../fonts/vazir/Vazir.eot');
+  src: url('../fonts/vazir/Vazir.eot?#iefix') format('embedded-opentype'),
+       url('../fonts/vazir/Vazir.woff2') format('woff2'),
+       url('../fonts/vazir/Vazir.woff') format('woff'),
+       url('../fonts/vazir/Vazir.ttf') format('truetype');
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: Vazir;
```

Comment:
I might be wrong as my CSS knowledge is limited but I think having many different font-face with the same font-family value and just changing the font-weight is not effective and If we want to include the weight we should also change the font-family accordingly. for example `Vazir-Bold`. 
I think we need only the normal font-weight for the font face definition.

Reply:
It is effective according to font-weight of rendered element:

![image](https://user-images.githubusercontent.com/5158436/58747441-94e18a00-8480-11e9-9319-197dc3e2b82c.png)

![image](https://user-images.githubusercontent.com/5158436/58747449-ae82d180-8480-11e9-95fe-3493f7cda479.png)

Reply:
I see, thanks for the explanation.

Reply:
Yes, you can do it, but it's not best practice.
it's better different font-face for all font weight